content/zh-cn/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 官方 CVE 订阅源
+weight: 25
+outputs:
+  - json
+  - html 
```

Comment:
We shouldn't produce a JSON output.

@PushkarJ you might like to add a note to the front matter of the English original, to explain that to localization teams.

Reply:
Noted. Could I simply remove the `outputs` lines from the localized version?
```
outputs:
  - json
  - html 
```

Reply:
Yep, that fixed it.